pkgs/by-name/br/bruno/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
```

Comment:
Nit: These parameters are commonly formatted this way

```
{ a
, b
, c
```

Reply:
I formatted this with Alejandra instead of nixpkgs-fmt by accident, reformatted.